public/app/features/explore/TraceView/createSpanLink.test.ts

```diff
@@ -172,7 +172,7 @@ describe('createSpanLinkFactory', () => {
       expect(createLink).toBeDefined();
       const links = createLink!(createTraceSpan());
 
-      const linkDef = links?.logLinks?.[0];
+      const linkDef = links?.otherLinks?.[0];
```

Comment:
I will build out the tests for the additions to `createSpanLink` once we have a consensus on what to do.